torchao/prototype/calibration_flow/static_quant.py

```diff
@@ -0,0 +1,110 @@
+"""
```

Comment:
no test?

Reply:
this is a demo for how static quantization can be implemented, instead of an API, should this be moved to some where else like `tutorials` folder?

Reply:
Yeah perhaps docs is a better location for now